### DISCUSSION TITLE
Add HVAC modbus support for fan settings

### DISCUSSION THREAD

**Discussion by wrongisthenewright** [home-assistant/frontend#17555] [Configuration]
I think it will be useful to add to the modbus climate integration the same features present in the MQTT climate entity, in particular the fan settings that's are missing in the modbus version.
I think it will be relatively simply to implement, given that it will add reading and writing only other register apart the ones already implemented.

**Comment by karwosts** [comment]
I think this doesn't seem like a frontend issue?

If you just want to make general HA feature requests, the place to do so is here: https://community.home-assistant.io/c/feature-requests/13

**Reply by wrongisthenewright** [reply]
Ok my bad. I'll request there.
Thank you
